packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -557,7 +557,7 @@ const MultiSelect = (props: Props) => {
                             onBlur={onOpenerBlurValidation}
                             disabled={isDisabled}
                             ref={handleOpenerRef}
-                            text={menuText}
+                            content={menuContent}
                             opened={open}
```

Comment:
Do we need to apply the `aria-label` to the DropdownOpener too for when a custom opener is provided? Here's an [example of using a custom opener](https://5e1bf4b385e3fb0020b7073c-kblofxoolb.chromatic.com/?path=/story/packages-dropdown-singleselect--custom-opener)!

Reply:
Yes we do! That might be a good huddle topic, since I wasn't exactly sure how to make that work in my first attempt. I have some stashed changes I could pick up, but I figured they could happen in a follow-up.

Reply:
This is fixed and added to the PR!